src/plots/mapbox/layout_attributes.js

```diff
@@ -117,7 +137,7 @@ var attrs = module.exports = overrideAll({
                 'Sets the source data for this layer (mapbox.layer.source).',
                 'Source can be either a URL,',
                 'a geojson object (with `sourcetype` set to *geojson*)',
-                'or an array of tile URLS (with `sourcetype` set to *vector*).'
+                'or an array of URLs (with `sourcetype` set to *vector* or *raster*).'
```

Comment:
what does `sourcetype` have to be if `source` is a URL?

Reply:
And actually I would leave the word "tile" in there, as it's not super clear what the URL should point to otherwise. We should try to say what the non-array URL should point to also: an image? a GeoJSON file?

Reply:
Now this is:

https://github.com/plotly/plotly.js/blob/29c2bf9ce97403ae844ac0c27c3be3d67da1948d/src/plots/mapbox/layout_attributes.js#L137-L142